README.md

```diff
@@ -0,0 +1,62 @@
+![logo](https://upload.wikimedia.org/wikipedia/commons/b/b5/Kore_2g_logo.png)
+
+* OpenKore is an custom client and intelligent automated assistant for Ragnarok Online.
+* It is a **free**, open source and cross-platform program (Linux and Windows are supported).
+
+## Prerequisites
+
+To run OpenKore you will need:
+1. [Download and Install Perl](http://openkore.com/index.php/How_to_run_OpenKore#Perl) (You can run start.exe or wxstart.exe if you use microsoft windows)
+
+## Quickstart
+
+1.  [Download OpenKore](https://github.com/OpenKore/openkore/archive/master.zip) and extract it.
+2.  Configure OpenKore: [documentation](http://openkore.com/index.php/Category:Control).
+3.  [Compile fields](http://openkore.com/index.php/FLD_Creation_Guide) to add fields to openkore.
+
+## Things to know
+
+* Before submitting an issue, ensure you are on the latest release.
+* Check if an issue already exists for your request.
+* Please post in English if possible.
+* Please use the issue template, please post information about the server and your configuration.
+* Remember, developers are doing this on free time.
+
+### Status of botting on Official Servers
+
+| Server | Description | Status | Supporter |
+| --- | --- | --- | --- |
+| `iRO` | International(US) RO | Botable, Weekly Update | allanon |
+| `twRO` | Taiwanese RO | Not working | windhamwong |
+| `bRO` | Brazilian RO | Botable, Weekly Update | Henrybk, lututui, alisonrag |
+| `idRO` | Indonesian RO | working only RE_server | sctnightcore(temp)
+| `fRO` | Europe(french) RO | working | sctnightcore | 
+| `mRO` | Maylisian RO | Need Verification | N/A |
```

Comment:
How about mysgRo? It's My&Sg.
Btw, it's Malaysia/Malaysian not, Maylisian (Not sure sarcasm or not) 😐